### DISCUSSION TITLE
Syntax for Magick.NET reference building with MCS

### DISCUSSION THREAD

**Discussion by RandomCoding-74** [dlemstra/Magick.NET#1756] [Help]
When trying to build my .cs with mcs, I keep getting could not be found errors. Ive learnt that mcs needs a specific reference, such as /reference:System.Drawing. 
How do I write the reference for Magick.NET? Ive tried all sorts including /reference:Magick.NET to /reference:Magick.NET.Core.dll. Using VScodium on Ubuntu 22.04, working on a mono console project in C#

Essentially Im here learning (trying) to make a small app to recognise images on screen, and System.Drawing doesnt want to work out windows. Any ideas welcome, and thanks

**Comment by dlemstra** [comment]
Why are you using `mcs` and not `dotnet`? You should be able to use that on a Linux platform?

**Comment by RandomCoding-74** [comment]
Mostly because thats whats in the GTK tutorials. I dont know much about the different compilers or how they work yet. For now, Ive started building straight in VScode with its .NET which gives me a linux executable.

**Comment by dlemstra** [comment]
I would advise you to read up on how you can use the `dotnet` command to do this. Microsoft is moving forward on that path and I don't think you should me using `msc`.

**Reply by RandomCoding-74** [reply]
Alright will do, thanks
